### ISSUE TITLE
0,0 lat, lon for some trips

### ISSUE THREAD

**Issue by mshodge** [planarnetwork/transxchange2gtfs#9]
Hi there, I've been using the tool to convert the Welsh bus data to GTFS. It seems though that some stops are being assigned a 0,0 lat, lon. For the entire Welsh dataset this is only 13.

I've attached an example XML file where Moor Lane is given a 0,0 lat, lon, yet in the XML it has:

```
    <AnnotatedStopPointRef>
      <StopPointRef>5230CGX16334</StopPointRef>
      <CommonName>Moor Lane</CommonName>
      <Location id="L36">
        <Longitude>-4.079770629</Longitude>
        <Latitude>52.415492344</Latitude>
      </Location>
    </AnnotatedStopPointRef>
```

Thanks!

[PadarnCars1_TXC_2019104-0106_CGAT022.xml.zip](https://github.com/planarnetwork/transxchange2gtfs/files/2888587/PadarnCars1_TXC_2019104-0106_CGAT022.xml.zip)

[out-gtfs.zip](https://github.com/planarnetwork/transxchange2gtfs/files/2888589/out-gtfs.zip)

**Comment by linusnorton** [comment]
Looking at the implementation it appears they are hardcoded to 0,0 - I will take a look at what is involved in implementing this correctly.

**Comment by mshodge** [comment]
Thanks! Was just a flag raised by someone I gave some analysis to using OTP, OTP correctly gave back the durations from the timetables but returned a massive distance value because of the 0,0.

**Comment by linusnorton** [comment]
Just out of interest where did this file come from? I get the data from traveline and I've never seen the stop Location.Latitude/Longitude used.

**Comment by linusnorton** [comment]
A fix for this has been release with version 1.2.0

**Comment by mshodge** [comment]
Thanks! This came from Travelline Cymru (through the Welsh Government).
